### PR TITLE
update two test cases for review firstly

### PR DESCRIPTION
**Pull request by danster**
this PR has labeled the corresponding test case in comments of cases, and taken off the blank line issue.

### CODE REVIEW

File: ecstest/testcases/dataplane/object_list_test_extend.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    def _create_bucket(self, bucket_name=None):
+        """
+        To create bucket with bucket_name
+        """
+        if bucket_name is None:
+            bucket_name = bucketname.get_unique_bucket_name()
+
+        logger.debug("Create bucket: %s", bucket_name)
+        bucket = self.data_conn.create_bucket(bucket_name)
```
Comment (MingatRubicon) [inline ecstest/testcases/dataplane/object_list_test_extend.py:49]: assert to check if this is a valid bucket object,

Comment (danster) [inline ecstest/testcases/dataplane/object_list_test_extend.py:49]: I have taken off this issue, please review, thank you.

File: ecstest/testcases/dataplane/object_list_test_extend.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            key.set_contents_from_string(s)
+
+    @triage
+    # corresponding tests: test_bucket_list_distinct().
+    def test_object_list_from_distinct_bucket(self):
+        """
+        operation: list
+        assertion: distinct buckets have different contents
+        """
+        bucket1 = self._create_bucket()
```
Comment (MingatRubicon) [inline ecstest/testcases/dataplane/object_list_test_extend.py]: these two buckets will not be cleared out. need to clean up

File: ecstest/testcases/dataplane/object_list_test_extend.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        bucket2 = self._create_bucket()
+
+        key = bucket1.new_key('asdf')
```
Comment (MingatRubicon) [inline ecstest/testcases/dataplane/object_list_test_extend.py]: we have method to get unique keys

File: ecstest/testcases/dataplane/object_list_test_extend.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        Populate a (specified or new) bucket with objects with
+        specified names (and contents identical to their names).
+        """
+        for s in keys:
+            key = self.bucket.new_key(s)
+            key.set_contents_from_string(s)
+
+    @triage
+    # corresponding tests: test_bucket_list_distinct().
+    def test_object_list_from_distinct_bucket(self):
```
Comment (MingatRubicon) [inline ecstest/testcases/dataplane/object_list_test_extend.py:66]: add a comment to indicate which test case (https://github.com/ceph/s3-tests/blob/master/s3tests/functional/test_s3.py)  was ported from.

File: ecstest/testcases/dataplane/object_list_test_extend.py (hunk 3 above)
Comment (danster) [inline ecstest/testcases/dataplane/object_list_test_extend.py]: have resolved, please review, thank you.

File: ecstest/testcases/dataplane/object_list_test_extend.py (hunk 4 above)
Comment (danster) [inline ecstest/testcases/dataplane/object_list_test_extend.py:66]: have added the indication.

**Pull request merged**